test/unit/connectors/BindingTariffFilestoreConnectorSpec.scala

```diff
@@ -57,7 +58,7 @@ class BindingTariffFilestoreConnectorSpec
           .willReturn(
             aResponse()
               .withStatus(Status.ACCEPTED)
-              .withBody(fromFile("test/resources/binding-tariff-filestore_upload-response.json"))
+              .withBody(fromFile("test/unit/resources/binding-tariff-filestore_upload-response.json"))
```

Comment:
Are these resources not better placed in the util directory? So they can be used with ITs and Unit Tests

Reply:
If they are only used in unit tests at the moment then they are fine where you have put them, your call 👍

Reply:
on the next PR we could move it. This PR was purely for the Wiremock

Reply:
We should not have non-code files mixed with code files.
The non-code files should be inside `test/resources`

Reply:
Done